.github/workflows/wheels.yml

```diff
@@ -106,6 +106,10 @@ jobs:
           # MACOS linker doesn't support stripping symbols
           CIBW_ENVIRONMENT_MACOS: CFLAGS='-std=c99 -fno-strict-aliasing'
                                   OPENBLAS64_=/usr/local
+          # TODO: Add universal2 wheels, we need to fuse them manually
+          # instead of going through cibuildwheel
```

Comment:
Could you add the reason for that, or a link to an issue/PR where that is discussed?